src/actions/events.js

```diff
@@ -0,0 +1,25 @@
+import { SAVE_POST, FETCH_POSTS } from '../ActionsTYPES/TYPES'
+
+
+export const savePost = (post) =>
+    (dispatch, getState, getFirebase) => {
+        const firebase = getFirebase()
+        firebase.database().ref(`groups/${post.groupTitle}/posts/${post.postIndex}`)
+            .set(post)
+            .then(() => {
+                dispatch({ type: SAVE_POST, payload: 'Success' })
```

Comment:
'success' ?  look at my reducer how it should be, if you want to update just status, u need to create property inside reducer

Reply:
I will work on it